packages/design-system/src/index.ts

```diff
@@ -1,182 +1,43 @@
-export type * from './components/Badge';
+export * from './components/Accordion';
```

Comment:
I see only one potential issue but maybe no risk.
what's happened if two folder export the same thing (a type, a component or something else with the same name)

Reply:
In this case the last export is the one.
If you have named two different things the same it means naming of one of them is bad or not enough hidden.

I had the same question in mind but different: today most of the componetn related to form are inside folders. to access them you need to go throw, Form.Checkbox but you also have Form.Input.Checkbox

I don't know why we do not have just {Checkbox} and all the rest removed

Even worse: Checkbox is the checkbox + label. This is in my mind a bigger issue.